docs/reference/reference-training/experiment-config-reference.rst

```diff
@@ -1339,6 +1339,11 @@ The ``pbs`` section specifies configuration options applicable when the cluster
             - -p1000
             - -PMyProjectName
 
+   Requesting of resources and job placement may be influenced through use of ``-l``, however chunk
```

Comment:
I think that an example showing which "-l" options we would ignore would be helpful to the reader.

Reply:
Example added.